Wordle.Api/Wordle.Api/appsettings.json

```diff
@@ -8,6 +8,6 @@
   "AllowedHosts": "*",
   "UseSwagger": "true",
   "ConnectionStrings": {
-    "DefaultConnection": "Data Source=(LocalDb)\\MSSQLLocalDB;Initial Catalog=WordleDev2023;Integrated Security=true;"
+    "DefaultConnection": "Server=localhost;User Id=sa;Password=myPassword123*;Initial Catalog=calvarez-assignment4-db;Trusted_Connection=False;TrustServerCertificate=True;Encrypt=True"
```

Comment:
Are you on a Mac or? I just don't get why you would change this. When I try to build the API, it doesn't work but I'm guessing this works for you otherwise you wouldn't have changed it.

Reply:
yes i am on a mac so i have to use a docker container and localhost rather than localdb.